lib/loopback-proxy-app/application.ts

```diff
@@ -16,14 +17,20 @@ export class LoopbackProxyApplication extends BootMixin(RepositoryMixin(RestAppl
     // Set up the custom sequence
     this.sequence(LabShareSequence);
 
+    this.bind(AuthenticationBindings.AUTH_CONFIG).to({
+      authUrl: options?.services?.auth?.url || options?.auth?.url || 'https://a.labshare.org/_api',
```

Comment:
is it possible to keep the hardcoded string value in some config file?

Reply:
@labshare/services is a module and doesn't have config file of its own. 
It's loading these config values from the application config file. The hardcoded string simply provides a fallback value. 
I can simply remove it from here to make it require setting these values in the config file.

Reply:
@dkrantsberg maybe removing it would be a good idea

Reply:
no keep the default value ,maybe you can declare a constant and set the default value .